core/domain/rights_manager_test.py

```diff
@@ -872,3 +980,151 @@ def test_moderator_can_unpublish_published_exploration(self):
         self.assertTrue(rights_manager.check_can_unpublish_activity(
             self.moderator,
             rights_manager.get_exploration_rights(self.published_exp_id)))
+
+
+class ActivityRightsTests(test_utils.GenericTestBase):
+
+    def setUp(self):
+        super(ActivityRightsTests, self).setUp()
+        self.signup(self.OWNER_EMAIL, self.OWNER_USERNAME)
+        self.owner_id = self.get_user_id_from_email(self.OWNER_EMAIL)
+        self.owner = user_services.UserActionsInfo(self.owner_id)
+        self.signup(self.VIEWER_EMAIL, self.VIEWER_USERNAME)
+        self.viewer_id = self.get_user_id_from_email(self.VIEWER_EMAIL)
+        self.viewer = user_services.UserActionsInfo(self.viewer_id)
+
+        self.exp_id = 'exp_id'
+        self.save_new_valid_exploration(self.exp_id, self.owner_id)
+        rights_manager.publish_exploration(self.owner, self.exp_id)
+        self.activity_rights = rights_manager.get_exploration_rights(
+            self.exp_id)
+
+    def test_validate_community_owned_explorations(self):
```

Comment:
Nit: explicitly assign an owner here to improve the readability of a test.

Reply:
This is for Community-owned explorations. It can have no owner